test/io/testfilewriter.py

```diff
@@ -20,15 +20,30 @@
 
 class WriterTest(unittest.TestCase):
 
+    def setUp(self):
+        self.MissingAttributeError = cclib.io.filewriter.MissingAttributeError
```

Comment:
If you want to make an alias for an exception in another module, better at module level. But I also see nothing wrong with using the entire path.